Helm/opensearch/values.yaml

```diff
@@ -111,15 +122,17 @@ resources:
     cpu: "1000m"
     memory: "100Mi"
 
-initResources: {}
+initResources:
+  {}
```

Comment:
IDE auto indenting?

Reply:
Certainly was, removed the egregious examples, left the expansion of the array in the config file to new lines, if you would prefer I can revert that as well.